app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/subscription/RecipientDetails.scala

```diff
@@ -16,16 +16,20 @@
 
 package uk.gov.hmrc.eoricommoncomponent.frontend.domain.subscription
 
+import play.api.i18n.Messages
 import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.eoricommoncomponent.frontend.models.{Journey, Service}
+import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName
 
 case class RecipientDetails(
   journey: Journey.Value,
   service: String,
+  serviceName: String,
   recipientEmailAddress: String,
   recipientFullName: String,
   orgName: Option[String],
-  completionDate: Option[String] = None
+  completionDate: Option[String] = None,
+  languageCode: Option[String] = None
```

Comment:
Should we have some default value? Or default value will be in the backend, so if this is not specified we will have English?

Reply:
Yes.  The BE will treat None as English.